lib/Layout/Resolver/Form/ConditionType/Mapper/ResourceType.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Layouts\Sylius\Layout\Resolver\Form\ConditionType\Mapper;
+
+use Netgen\Layouts\Layout\Resolver\Form\ConditionType\Mapper;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+
+use function str_replace;
+use function ucfirst;
+
+final class ResourceType extends Mapper
+{
+    /**
+     * @param array<string, string> $allowedResources
+     */
+    public function __construct(private readonly array $allowedResources) {}
+
+    public function getFormType(): string
+    {
+        return ChoiceType::class;
+    }
+
+    public function getFormOptions(): array
+    {
+        return [
+            'choices' => $this->getResourceTypeList(),
+            'choice_translation_domain' => false,
+            'multiple' => true,
+            'expanded' => true,
+        ];
+    }
+
+    /**
+     * @return array<string, string>
+     */
+    private function getResourceTypeList(): array
+    {
+        $resourceTypeList = [];
+
+        foreach ($this->allowedResources as $type) {
+            $resourceTypeList[$this->humanizeType($type)] = $type;
```

Comment:
This should go through `choice_label` option, e.g.

```
'choices' => $this->allowedResources,
'choice_label' => fn (string $type): string => $this->humanizeType($type),
```

Reply:
Fixed in https://github.com/netgen-layouts/layouts-sylius/pull/11/commits/93ab79cc5e0f027f9222331aa0551e4358fa80db